bcda/cli_test.go

```diff
@@ -535,7 +535,7 @@ func (s *CLITestSuite) TestSortCCLFFiles() {
 	cclflist = cclfmap["A0001_18"]
```

Comment:
I think we need the time formatting up on line 522 also (GitHub isn't letting me put a comment there)